boards/holybro/durandal-v1/init/rc.board_defaults

```diff
@@ -22,6 +22,9 @@ then
 fi
 unset BL_FILE
 
+# Enable IMU thermal control
+param set SENS_EN_THERMAL 1
```

Comment:
May want to make this an AUTOCNF thing to give users the flexibility to disable heater functionality. Right now, this force-enables it every boot.

Thoughts @dagar?

Reply:
Ha, I was just about to suggest that.

Reply:
Is there an update coming?

Reply:
Is there a change coming?